src/common/kits/EventManager.ts

```
@@ -0,0 +1,50 @@
+// Event 管理器
+class EventManager {
+  static instance: null | EventManager = null;
+  private eventList: { [key: string]: Function[] } = {};
+
+  static getInstance() {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)